### PR TITLE
experimental network ui

### PR DESCRIPTION
**Pull request by mavenugo**
Based on various discussions and inputs, we have a simplified proposal to address the network UI
separate from services ui. This PR brings in the network ui & api provided by libnetwork and hooking it
to be part of docker.
All of these are defined under experimental tag in order to get user feedback before making it officialy into the docker ui chain.

### CODE REVIEW

**Comment by icecrime** [comment]
Review this this morning with Madhu. I think the integration points for both CLI commands and API endpoints and clean and simple (directly goes to libnetwork, and it's still experimental).

The one tricky thing is the dance around `--net` modes, but that constraining stems from our requirement of backward compatibility. We shouldn't be changing the meaning of `--net=bridge`, and yet we need to allow changing the default behavior to something fancier (e.g., multihost) using a daemon-wide flag.

So LGTM for me.

**Comment by calavera** [comment]
LGTM. Moving to docs review.

**Comment by moxiegirl** [comment]
I don't have scope today for a full doc review. This is in experimental I see. So, let's merge it.  Just to remind everyone, experimental "docs" live in the repo and doesn't go out to docs.docker.com

**Comment by cpuguy83** [comment]
#boom

File: experimental/networking.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+
+Currently the only way this network can be used to connect container is via default network-mode.
+Docker daemon supports a configuration flag `--default-network` which takes configuration value of format `NETWORK:DRIVER`, where,
+`NETWORK` is the name of the network created using the `docker network create` command and 
+`DRIVER` represents the in-built drivers such as bridge, overlay, container, host and none. or Remote drivers via Network Plugins.
+When a container is created and if the network mode (`--net`) is not specified, then this default network will be used to connect
+the container. If `--default-network` is not specified, the default network will be the `bridge` driver.
+
+Send us feedback and comments on [#](https://github.com/docker/docker/issues/?),
```
Comment (thaJeztah) [inline experimental/networking.md]: @mavenugo can you open an issue and create a follow-up PR to fill in the right link to that issue? Thanks!

**Pull request merged**